thirdeye/thirdeye-frontend/tests/integration/pods/components/stats-cards/component-test.js

```diff
@@ -0,0 +1,67 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('stats-cards', 'Integration | Component | stats cards', {
+  integration: true
+});
+
+const CARD = '.te-horizontal-cards__card';
+const TITLE = `${CARD}-title`;
+const DESCRIPTION = `${CARD}-text`;
+const NUMBER = `${CARD}-number`;
+
+test('it renders', function(assert) {
+  const stats = [
+    ['Number of anomalies', 'total anomalies', 10],
+    ['Response Rate', 'description of response', 0.9],
+    ['Precision', 'description of precision', 1]
+  ];
+  this.setProperties({ stats });
+
+  this.render(
+    hbs`{{stats-cards
+          stats=stats}}`
+  );
+
+  // Testing titles of all cards
+  assert.equal(
+    this.$(TITLE).get(0).innerText,
```

Comment:
nit: each time we do `$()`, we're essentially using jquery to grab a dom object (not performant). 
Let's refactor this so that we only do that once per selector
```
const $title = this.$(TITLE);
const $description = this.$(DESCRIPTION)
const $number = this.$(NUMBER)

// and in your assert :
assert.equal($title.get(1).innerText .....

```